addon-test-support/index.js

```diff
@@ -1,10 +1,54 @@
 import { find, triggerEvent } from '@ember/test-helpers';
 import { assert } from '@ember/debug';
 
-export async function upload(selector, file) {
+export async function upload(selector, ...files) {
   let input = find(selector);
   assert(`Selector '${selector}' is not input element.`, input.tagName === 'INPUT');
-  assert(`File must be instance of File/Blob type`, file instanceof Blob);
+  assert(
+    `All files must be instances of File/Blob type`,
+    files.every((file) => file instanceof Blob)
+  );
 
-  return await triggerEvent(input, 'change', { files: [file] });
+  return triggerEvent(input, 'change', { files });
+}
+
+export async function dragAndDrop(selector, ...files) {
```

Comment:
I've kind of made a mess in here. Difficult to have a nice helper API and not duplicate a lot of code.

How much of this stuff do you think we should make available to users of the addon?

Reply:
What do you mean by mess? Looks clean and well structured to me. :+1:

Reply:
I guess I just mean the "boilerplate code" that is repeated.